BuildingSync.xsd

```diff
@@ -14687,6 +14693,50 @@
       </xs:restriction>
     </xs:simpleType>
   </xs:element>
+  <xs:element name="eGRIDSubregionCodes">
+    <xs:annotation>
+      <xs:documentation>The eGRID (Emissions and Generation Resource Database) subregion codes associated with the data being described.</xs:documentation>
+    </xs:annotation>
+    <xs:complexType>
+      <xs:sequence>
+        <xs:element name="eGRIDSubregionCode">
```

Comment:
I think we need a maxOccurs="unbounded" here, assuming we want to allow more than one

Reply:
is that not covered on this line: https://github.com/BuildingSync/schema/pull/387/files#diff-64393aee98cd0f73402be30b4800f2ac497b4a5a06a591db6b3c207ca023e8ebR521

Reply:
@nllong sorry which line is that? It's just linking to a file diff

@JieXiong9119 
> The default value for both the minOccurs and the maxOccurs attributes is 1

https://www.w3.org/TR/xmlschema-0/#OccurrenceConstraints

Reply:
Revised.